jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -142,6 +142,15 @@ def search(self, error_on_fail=False):
                                       query_region_result['dec'],
                                       unit='deg')
 
+        elif self.catalog_selected == 'Gaia':
+            from astroquery.gaia import Gaia
+
+            Gaia.ROW_LIMIT = 5_000
+            sources = Gaia.query_object(skycoord_center, radius=3 * u.arcmin,
```

Comment:
We should replace `radius=3 * u.arcmin` with the half-width of a loaded image or the visible FOV.

Reply:
SDSS seems to still do the same, but agreed that we should change it to be more flexible in both